src/mlpack/methods/ann/activation_functions/selu_function.hpp

```diff
@@ -0,0 +1,164 @@
+/**
+ * @file selu_function.hpp
+ * @author Dakshit Agrawal
+ *
+ * Definition and implementation of the SELU function as introduced by
+ * Klambauer et. al. in Self Neural Networks.  The SELU activation
+ * function keeps the mean and variance of the input invariant.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @article{Klambauer2017,
+ *   author  = {Gunter Klambauer and Thomas Unterthiner and
+ *              Andreas Mayr},
+ *   title   = {Self-Normalizing Neural Networks},
+ *   journal = {Advances in Neural Information Processing Systems},
+ *   year    = {2017}
+ * }
+ * }
```

Comment:
Can we also put the reference to the paper in the class documentation (below) also please?  A user may not find this documentation if they are, e.g., looking at Doxygen output or something.

Reply:
Sure I will do that.